shared/util/errors.js.flow

```diff
@@ -0,0 +1,26 @@
+// @flow
+// Proxy so we don't get import loops. We need rpc gen but it also needs us
```

Comment:
`niceError` is the only fn that uses RPCTypes...perhaps move that into its own file instead?